test/utils/conditions.go

```diff
@@ -23,7 +23,7 @@ import (
 	podutil "k8s.io/kubernetes/pkg/api/v1/pod"
 )
 
-type ContainerFailures struct {
+type containerFailures struct {
```

Comment:
Could you explain the reason for un-export this struct?

Reply:
Hi @draveness

Thank for review my first pull request for k8s.
ContainerFailures struct called only in this file test/util/conditions.go

“s/newcert/newCert”

The above modification is also based on this reason.

Best wishes

Reply:
> ContainerFailures struct called only in this file test/util/conditions.go

SGTM

> “s/newcert/newCert”
> 
> The above modification is also based on this reason.

Please make the "c" to uppercase, which is not the first letter in the function.